common/v2/features/NoAccounts/NoAccounts.tsx

```diff
@@ -94,23 +95,33 @@ const ButtonGroup = styled.div`
   margin-top: 20px;
 `;
 
-export const NoAccounts = () => (
-  <NoAccountsContainer>
-    <NoAccountsContent>
-      <ImgIcon src={sadWallet} />
-      <Header>{translate('NO_ACCOUNTS_HEADER')}</Header>
-      <Description>{translate('NO_ACCOUNTS_DESCRIPTION')}</Description>
-      <ButtonGroup>
-        <Link to={ROUTE_PATHS.ADD_ACCOUNT.path}>
-          <WhiteButtonFirst secondary={true}>{translate('ADD_EXISTING_ACCOUNT')}</WhiteButtonFirst>
-        </Link>
-        <Link to={ROUTE_PATHS.SETTINGS_IMPORT.path}>
-          <PrimaryButton secondary={true}>{translate('SETTINGS_IMPORT_HEADING')}</PrimaryButton>
+export const NoAccounts = () => {
+  useEffect(() => {
```

Comment:
We have `useFirstMount` available under `v2/vendor`

Reply:
How would I use this, exactly?